Simplenote/SPAppDelegate.m

```diff
@@ -121,15 +119,23 @@ - (void)setupThemeNotifications
     [nc addObserver:self selector:@selector(themeDidChange) name:SPSimplenoteThemeChangedNotification object:nil];
 }
 
+- (void)setupStorage
+{
+    StorageSettings *storageSettings = [StorageSettings new];
+    SharedStorageMigrator *storageMigrator = [[SharedStorageMigrator alloc] initWithStorageSettings:storageSettings];
+    [storageMigrator performMigrationIfNeeded];
+
+    self.coreDataManager = [[CoreDataManager alloc] initWithStorageSettings:storageSettings];
+}
```

Comment:
I think this can be moved to swift extension 🤔

Reply:
+1 Swift please (and it should prob. be two methods, `ensureStorageIsShared` maybe, and `setupCoreData` or so).

Reply:
I moved the setup to the swift file.  To be able to setup the property in the swift file I had to remove the read only flag from SPAppDelegate.h declaration of the core data manager.

Shouldn't be an issue but wanted to mention it.